notebook/tests/test_files.py

```diff
@@ -95,7 +95,7 @@ def test_contents_manager(self):
 
         r = self.request('GET', 'files/test.txt')
         self.assertEqual(r.status_code, 200)
-        self.assertEqual(r.headers['content-type'], 'text/plain; charset=UTF-8')
```

Comment:
It looks like the tests are passing on Travis... It looks good to me. What says @minrk?